pkg/virt-handler/cgroup/util.go

```diff
@@ -130,6 +130,13 @@ func GenerateDefaultDeviceRules() []*devices.Rule {
 			Permissions: permissions,
 			Allow:       toAllow,
 		},
+		{ // /dev/vhost-vsock
```

Comment:
`/dev/vhost-vsock` should not be present in default devices rules, since it is enabled via featureGate

Reply:
Done